src/Yoti.Auth/YotiClientEngine.cs

```diff
@@ -25,13 +25,6 @@ public YotiClientEngine(IHttpRequester httpRequester, HttpClient httpClient)
 #endif
         }
 
-        public ActivityDetails GetActivityDetails(string encryptedToken, string sdkId, AsymmetricCipherKeyPair keyPair, Uri apiUrl)
```

Comment:
Is it convention to suffix async methods with `...Async`? If not, could we make `GetActivityDetails` async and remove `GetActivityDetailsAsync`?

Reply:
Yes it's a .NET convention.

From <https://docs.microsoft.com/en-us/dotnet/csharp/async>: 

> You should add "Async" as the suffix of every async method name you write.
> This is the convention used in .NET to more-easily differentiate synchronous and asynchronous methods. Note that certain methods which aren’t explicitly called by your code (such as event handlers or web controller methods) don’t necessarily apply. Because these are not explicitly called by your code, being explicit about their naming isn’t as important.

There is some disagreement([1](https://stackoverflow.com/a/47810905/1014632) | [2](https://docs.particular.net/nservicebus/upgrades/5to6/async-suffix)) on this if your code doesn't contain both async and sync methods (which is the case here), and it's also not in a public class, but in general I think it would be best to go with the convention.